### ISSUE TITLE
no-unused-vars when assigning and using a store in a different file

### ISSUE THREAD

**Issue by manuel3108** [sveltejs/eslint-plugin-svelte3#79]
Let's say we have a store:
```js
import { writable } from 'svelte/store';

export const name = writable('asdf');
```

an `index.svelte`:
```svelte
<script>
    import { name } from '../stores/name';

    function onClick() {
        $name = 2;
    }
</script>

<button on:click={onClick}>Some useless html</button>
```

and an `about.svelte`:
```svelte
<script>
    import { name } from '../stores/name';
</script>

<h1>{$name}</h1>
```

Both svelte files are located in the `routes` directory.
Eslint is called from the `package.json` with `eslint ./src`

Relevant `package.json` info:
```json
  "devDependencies": {
    "eslint": "^7.16.0",
    "eslint-config-airbnb-base": "^14.2.1",
    "eslint-plugin-import": "^2.22.1",
    "eslint-plugin-svelte3": "^3.0.0",
    "sapper": "^0.28.0",
    "svelte": "^3.17.3"
  }
```

Expected result:
Var should be used.

Actual result:
Var is apparently unused.

Minimal reproduction template: https://github.com/manuel3108/sapper-eslint-issue

Are we doing anything wrong? Is this in expected behaviour?
Currently we are getting around with adding `// eslint-disable-next-line no-unused-vars` over such issues

**Comment by Conduitry** [comment]
This should be fixed in 3.2.0 by way of #108.

**Comment by JohnnyFun** [comment]
Confirmed fixed for me! Thanks!
